djangocms_page_meta/utils.py

```diff
@@ -128,6 +129,7 @@ def get_page_meta(page, language):
                     pass
             if not meta.image and pagemeta.image:
                 meta.image = pagemeta.image.canonical_url or pagemeta.image.url
+                meta.schemaorg_image = pagemeta.image
```

Comment:
I am not sure that we should use `pagemeta.image` in `meta.schemaorg_image` rather than `pagemeta.image.canonical_url or pagemeta.image.url` like we do for `meta.image`: shouldn't they be the same?

If they do are the same, can't we just set `meta.schemaorg_image = meta.image` at the bottom of this method?

Reply:
No, you're right. Looking at schema.org documentation, image itemprop must be used like this: `<meta itemprop="image" content="your-image-url">` . So, we should totally use an url there. I'll change as you suggested.

Reply:
:+1: